src/components/AppProvider/utilities/createAppProviderContext/tests/createAppProviderContext.test.tsx

```diff
@@ -85,36 +85,33 @@ describe('createAppProviderContext()', () => {
     expect(context).toEqual(mockContext);
   });
 
-  it('does not instantiate app bridge if server side rendering', () => {
-    mockIsServer(true);
+  it('initializes a noop app bridge if server side rendering', () => {
+    jest.spyOn(redirect, 'getWindow').mockReturnValueOnce(undefined);
```

Comment:
We should treat app-bridge as a blackbox and avoid reaching into its internals. I think we can just extend the `returns the right context with properties` with assertions for `dispatch`, `getState` et... App-bridge is responsible to handle the case when window is undefined - Polaris does not need to care about this anymore

Reply:
Ah, that makes things much easier, it also makes way more sense. Thanks!